source/FreeRTOS_UDP_IPv6.c

```diff
@@ -414,7 +414,10 @@ BaseType_t xProcessReceivedUDPPacket_IPv6( NetworkBufferDescriptor_t * pxNetwork
 
     configASSERT( pxNetworkBuffer != NULL );
     configASSERT( pxNetworkBuffer->pucEthernetBuffer != NULL );
-    UDPPacket_IPv6_t * pxUDPPacket_IPv6;
+    /* When refreshing the ARP/ND cache with received UDP packets we must be
+     * careful;  hundreds of broadcast messages may pass and if we're not
+     * handling them, no use to fill the cache with those IP addresses. */
+    UDPPacket_IPv6_t * pxUDPPacket_IPv6 = ( ( UDPPacket_IPv6_t* ) pxNetworkBuffer->pucEthernetBuffer );
```

Comment:
This one is needed for "ping winsim".
Issue is pxUDPPacket_IPv6 not being initialized.